tests/post-duplicator-test.php

```diff
@@ -7,11 +7,12 @@
 use WP_Post;
 use WP_User;
 use Yoast\WP\Duplicate_Post\Post_Duplicator;
+use Yoast\WPTestUtils\BrainMonkey\TestCase as BMTestCase;
 
 /**
  * Test the Post_Duplicator class.
  */
-class Post_Duplicator_Test extends TestCase {
+class Post_Duplicator_Test extends BMTestCase {
```

Comment:
Looking at where `sanitized_text_field` has expectations set, it doesn't really have any testing purposes and it was a bad substitute for a stub.
I'd say we should remove that expectation from `test_generate_copy_title` and switch to extending the same `TestCase` class here as well, what do you think @jrfnl?

Reply:
I'd be perfectly happy to make that change.

Reply:
FYI: change made in a separate new commit.